### DISCUSSION TITLE
support dacast push service?

### DISCUSSION THREAD

**Discussion by duckscorpion13** [HaishinKit/HaishinKit.swift#1024] [Q&A]
I use this website
https://app.dacast.com/
but auth always fail

NetConnection.Connect.Rejected
2022-07-05 10:38:52.852761+0800 rtmpsTest[17260:8091643] [tcp] tcp_output [C2.1:1] flags=[R.] seq=322489201, ack=426575500, win=2048 state=CLOSED rcv_nxt=426575500, snd_una=322489185

var uri: String? ="rtmp://[user]:[pwd]@2733421a-0fac-xxx-a137xxxxxd2d3.dacastmmd.pri.lldns.net/dacastmmd/"
var streamName: String? = "ea340648xxxxxxx374e265_4500"

**Comment by shogo4405** [comment]
Can I see. `Logboard.with(HaishinKitIdentifier).level = .trace` log?

How about this? Please remove last `/`.
```
var uri: String? ="rtmp://[user]:[pwd]@2733421a-0fac-xxx-a137xxxxxd2d3.dacastmmd.pri.lldns.net/dacastmmd"
var streamName: String? = "ea340648xxxxxxx374e265_4500"
```

**Reply by pyaephyoaung-pa** [reply]
I also face this issue NetConnection.Connect.Rejected without last "/". Could you please help?
